internal/service/ses/active_receipt_rule_set_data_source_test.go

```diff
@@ -50,3 +77,34 @@ data "aws_ses_active_receipt_rule_set" "test" {
 }
 `, name)
 }
+
+func testAccActiveReceiptRuleSetDataSourceConfig_noActiveRuleSet() string {
+	return `
+data "aws_ses_active_receipt_rule_set" "test" {}
+`
+}
+
+func testAccPreCheckUnsetActiveRuleSet(t *testing.T) {
+	conn := acctest.Provider.Meta().(*conns.AWSClient).SESConn
+
+	output, err := conn.DescribeActiveReceiptRuleSet(&ses.DescribeActiveReceiptRuleSetInput{})
+	if acctest.PreCheckSkipError(err) {
+		t.Skipf("skipping acceptance testing: %s", err)
+	}
+	if output == nil || output.Metadata == nil {
+		return
+	}
+	if err != nil {
+		t.Fatalf("unexpected PreCheck error: %s", err)
+	}
+
+	_, err = conn.SetActiveReceiptRuleSet(&ses.SetActiveReceiptRuleSetInput{
+		RuleSetName: nil,
+	})
```

Comment:
Passing a null `RuleSetName` is how the AWS docs suggest removing an active receipt rule set:
https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/ses#Client.SetActiveReceiptRuleSet